mlibrary/search
# Shelf browse carousel loading state

# Overview
Waiting for Shelf Browse to appear can take a while. This pull request creates a loading state while waiting for Catalog Browse data to be fetched.

## Testing
- Make sure the PR is consistent in these browsers:
- Run accessibility tests:
- Check out a catalog item with a call number. Do you see the loading state? Can you still successfully navigate the carousel?
